assets/js/base/components/label/index.tsx

```diff
@@ -6,8 +6,8 @@ import classNames from 'classnames';
 import type { ReactElement, HTMLProps } from 'react';
 
 interface LabelProps extends HTMLProps< HTMLElement > {
-	label?: string;
-	screenReaderLabel?: string;
+	label?: string | undefined;
```

Comment:
This is required because of the changes in #5312 where `"exactOptionalPropertyTypes": true` was added to `tsconfig.base.json`. The same applies for other instances where I added `| undefined` to optional props.